src/plugins/intel_gpu/src/kernel_selector/cl_kernels/select_gpu_ref.cl

```diff
@@ -4,23 +4,46 @@
 
 #include "include/batch_headers/fetch_data.cl"
 
-#define INPUT_0 input0[INPUT0_GET_INDEX_SAFE(b, f, y, x)]
-#define INPUT_1 input1[INPUT1_GET_INDEX_SAFE(b, f, y, x)]
-#define INPUT_2 input2[INPUT2_GET_INDEX_SAFE(b, f, y, x)]
+#if OUTPUT_DIMS == 5
+    #define INPUT_0 input0[INPUT0_GET_INDEX_SAFE(b, f, z, y, x)]
+    #if INPUT1_DIMS == 4
+        #define INPUT_1 input1[INPUT1_GET_INDEX_SAFE(b, f, y, x)]
```

Comment:
could you add test for such broadcasted case too?

Reply:
Updated as you commented.